src/authorization/handler.signUp.js

```diff
@@ -0,0 +1,16 @@
+const { encryptionPassword } = require('./cipher');
+const { hashPassword } = require('./hash.js');
+const PersonController = require('../PersonCRUD/person.controller')
+
+const handlerSignUp = async (body) => {
```

Comment:
```suggestion
const handlerSignUp = async (data) => {
```
Because of handler shoundn't know something about transport layer